src/Domain/Creators/CylindricalBinaryCompactObject.cpp

```diff
@@ -895,41 +902,47 @@ Domain<3> CylindricalBinaryCompactObject::create_domain() const {
     // map from the grid to inertial frame. No maps to the distorted frame
     grid_to_inertial_block_maps[0] =
         time_dependent_options_
-            ->grid_to_inertial_map<domain::ObjectLabel::None>(false);
+            ->grid_to_inertial_map<domain::ObjectLabel::None>(false, true);
+
+    if (include_outer_sphere_) {
```

Comment:
Doesn't this always have to be true?

Reply:
I think I thought what if there's a scenario where someone just wants the rotation map so they technically don't need to specify the outer sphere for that, buuuuuut that's not really useful

Reply:
After trying to take it out, I realized I kept this in for the tests, which tests both true and false for include_outer_sphere_. Should I change the test to make sure it errors if you use time-dependent maps without the outer sphere?

Reply:
Yeah no sense in trying to account for the special case of only a rotation. I believe the error in your constructor should catch this.